testing/mock/mock.go

```diff
@@ -94,6 +94,7 @@ type AppModule struct {
 	AppModuleBasic
 	scopedKeeper capabilitykeeper.ScopedKeeper
 	portKeeper   PortKeeper
+	BaseApp      MockIBCApp // base application of an IBC middleware stack
```

Comment:
I would change the naming here because i confused this with the SDK baseapp

Reply:
How about IBCApp?

Reply:
I like `IBCApp`